localization/english/wc_buildings_l_english.yml

```
@@ -22,11 +32,21 @@
  building_lordaeron_palace_01:1 "Lordaeron Palace"
  building_lordaeron_palace_01_desc:1 "The Palace of Lordaeron is the jewel of the Empire of Lordaeron and shows just how far humans have come. Its presence exudes an aura of prestige and with a hint of decadence."
  
+ building_type_nexus_01:0 "The Nexus"
+ building_type_nexus_01_desc:1 "The Nexus is an extensive series of caves and tunnels that riddles Coldarra."
```

```suggestion
 building_type_nexus_01_desc:1 "The Nexus is an extensive series of caves and tunnels that riddle Coldarra."
```
[Plural verbs](https://webapps.towson.edu/ows/moduleSVAGR1.jpg) usually don't have the _s_ added at the end.